rudder-web/src/main/scala/com/normation/rudder/web/components/popup/ModificationValidationPopup.scala

```diff
@@ -235,14 +217,17 @@ class ModificationValidationPopup(
   }
 
   private[this] val disabled = item match {
-    case Left(item) => ! item._4.isEnabled
```

Comment:
omg, tuple with different ordering, people who did this was mad (was a younger naive me, ha my lost youth ....)